src/cljs/main/broadfcui/common/entity_table.cljs

```diff
@@ -153,7 +153,9 @@
          (if (empty? entity-types)
            (on-done {:total-count 0 :filtered-count 0 :results []})
            (let [{:keys [page-number rows-per-page filter-text sort-column sort-order]} query-params
-                 entity-type (nth entity-types selected-filter-index)]
+                 entity-type (if (nil? selected-filter-index)
```

Comment:
i did initially try that, but then when the entites render, the first entity is selected even though it is not of the type expected. I think the behavior was not to have any selected.